List/index.html

```diff
@@ -0,0 +1,143 @@
+          <li><a href="../Input/index.html">Input</a></li>
+          <li><a href="../Grid/index.html">Grid</a></li>
+          <li><a href="../List/index.html">List</a></li>
+          <li><a href="../Modal/index.html">Modal</a></li>
+          <li><a href="../Navigation/index.html">Navigation</a></li>
+          <li><a href="../Rating/index.html">Rating</a></li>
+          <li><a href="../Text/index.html">Text</a></li>
+          <li><a href="../Toast/index.html">Toast</a></li>
+        </ul>
+      </nav>
+      <section>
+        <h3>Lists</h3>
+        <p>
+          Lists are continuous, vertical indexes of text or images. They are
+          composed of items containing primary and supplemental actions, which
+          are represented by icons and text.
+        </p>
+        <hr />
+        <h4>Simple List</h4>
+        <p></p>
+        <div class="container flex-container" style="justify-content: center">
+          <ul class="list">
+            <li class="btn-link">Home</li>
+            <li class="btn-link">SmartPhone</li>
+            <li class="btn-link">EarPhones</li>
+            <li class="btn-link">Watches</li>
+          </ul>
+        </div>
+        <div
+          class="container"
+          style="
+            height: 350px;
+            background-color: transparent;
+            padding-left: 0px;
+            padding-right: 0px; ;
+          "
+        >
+          <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28249%2C237%2C212%2C1%29&t=twilight&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=0px&dsblur=4px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=IBM+Plex+Mono&fs=14.5px&lh=154%25&si=false&es=2x&wm=false&code=%253Cul%2520class%253D%2522list%2522%253E%250A%2520%2520%253Cli%2520class%253D%2522btn-link%2522%253EHome%253C%252Fli%253E%250A%2520%2520%253Cli%2520class%253D%2522btn-link%2522%253ESmartPhone%253C%252Fli%253E%250A%2520%2520%253Cli%2520class%253D%2522btn-link%2522%253EEarPhones%253C%252Fli%253E%250A%2520%2520%253Cli%2520class%253D%2522btn-link%2522%253EWatches%253C%252Fli%253E%250A%253C%252Ful%253E%250A"
+            style="
+              width: 680px;
+              height: 350px;
+              border: 0;
+              transform: scale(1);
+              overflow: hidden;
+            "
+            sandbox="allow-scripts allow-same-origin"
+          >
+          </iframe>
+        </div>
+        <hr />
+        <h4>Stacked list</h4>
+        <p>Lists can be nested (list inside list):</p>
+        <div class="container flex-container" style="justify-content: center">
+          <ul class="list">
+            <li>Coffee</li>
+            <li>
+              Tea
+              <ul class="stacked">
+                <li>Black tea</li>
+                <li>Green tea</li>
+              </ul>
+            </li>
+            <li>Milk</li>
+          </ul>
+        </div>
+        <div
+          class="container"
+          style="
+            height: 350px;
```

Comment:
try not to use inline  styling

Reply:
> try not to use inline styling

I have  changed accordingly can you please approve it